data/transition-sites/gds_alphagov_content-planner-preview.yml

```diff
@@ -0,0 +1,8 @@
+---
+site: gds_alphagov_content-planner-preview
```

Comment:
Hasn't content-planner been removed everywhere? We [didn't redirect it](https://github.com/alphagov/transition-config/pull/1128) in the end in Production.